pages/common/virt-sparsify.md

```diff
@@ -0,0 +1,13 @@
+# virt-sparsify
+
+> libguestfs tool to to make a virtual machine drive images thin-provisioned.
+> Use only for offline machines to avoid data corruption.
+> Home page: <https://libguestfs.org/>.
```

Comment:
Same deal here as with the previous page. What about this?
```suggestion
> Make virtual machine drive images thin-provisioned.
> Use only for offline machines to avoid data corruption.
> More information: <https://libguestfs.org/>.
```

Reply:
I'd prefer to keep a libguestfs mention, cause it could help to match the utility with a distro package. But it is up to you.